qa/rpc-tests/feature_fedpeg.py

```diff
@@ -47,27 +49,57 @@ def __init__(self):
         self.setup_clean_chain = True
         self.num_nodes = 4
 
+    def add_options(self, parser):
+        parser.add_option("--parent_binpath", dest="parent_binpath", default="",
+                          help="Use a different binary for launching nodes")
+        parser.add_option("--parent_bitcoin", dest="parent_bitcoin", default=False, action="store_true",
+                          help="Parent nodes are Bitcoin")
+
     def setup_network(self, split=False):
 
         # Parent chain args
-        self.extra_args = [[
-            # '-printtoconsole',
-            '-validatepegin=0',
-            '-anyonecanspendaremine',
-            '-initialfreecoins=2100000000000000',
-        ]] * 2
-
-        self.nodes = start_nodes(2, self.options.tmpdir, self.extra_args[:2], chain='parent')
+        chain = None
+        if self.options.parent_bitcoin:
+            if self.options.parent_binpath == "":
```

Comment:
Shouldn't this be filtered in rpc-tests.py as well? This way a top-level arg there could trigger this, or not, safely.

Reply:
I don't understand your point..

Reply:
rpc-tests.py will run all tests in sequence. I'd rather catch this argument error up front instead of waiting for this to be run.

Reply:
Oh, you can pass these arguments to rpc-tests.py and they are globally set for all tests??

Reply:
That was my hope, yes

Should be able to just make it some global(?)

Reply:
Euhm, it seems that that's not possible anyway. If you run rpc-tests.py with those arguments, all other tests will complain that they don't know the arguments:
```
create_cache.py: error: no such option: --parent_bitcoin
```

Also, I think for now it makes more sense to only use those flags when running this test specifically..

Reply:
meh ok